<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlyFiClass</class>
 <widget class="QMainWindow" name="FlyFiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Floppy Orgel 3.0 Controller</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>441</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Debug</string>
    </property>
    <widget class="QTextEdit" name="edtDebug">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>251</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>331</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Error</string>
    </property>
    <widget class="QTextEdit" name="edtError">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>251</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnClearDebug">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>441</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearError">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>620</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>521</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial and MIDI ports</string>
    </property>
    <widget class="QPushButton" name="btnRefreshPorts">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial Port:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbSerialPorts">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maxVisibleItems">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbMidiPorts">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbBaudrate">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MIDI Port:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenSerial">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>90</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCloseSerial">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>90</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <zorder>btnCloseSerial</zorder>
    <zorder>label_2</zorder>
    <zorder>cmbSerialPorts</zorder>
    <zorder>label_3</zorder>
    <zorder>cmbMidiPorts</zorder>
    <zorder>cmbBaudrate</zorder>
    <zorder>label_4</zorder>
    <zorder>btnOpenSerial</zorder>
    <zorder>btnRefreshPorts</zorder>
    <zorder>label_2</zorder>
    <zorder>cmbSerialPorts</zorder>
    <zorder>label_3</zorder>
    <zorder>cmbMidiPorts</zorder>
    <zorder>cmbBaudrate</zorder>
    <zorder>label_4</zorder>
    <zorder>btnOpenSerial</zorder>
    <zorder>btnCloseSerial</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>521</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Playback Test</string>
    </property>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>218</x>
       <y>111</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>71</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPlay">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>111</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QSlider" name="sldFreq">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>71</y>
       <width>251</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximum">
      <number>60000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblFreq">
     <property name="geometry">
      <rect>
       <x>362</x>
       <y>71</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00 Hz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>39</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Drive:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive1">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive2">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive3">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive4">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive5">
     <property name="geometry">
      <rect>
       <x>172</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive6">
     <property name="geometry">
      <rect>
       <x>192</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive7">
     <property name="geometry">
      <rect>
       <x>212</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive8">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive9">
     <property name="geometry">
      <rect>
       <x>252</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive10">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive11">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive12">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive13">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive14">
     <property name="geometry">
      <rect>
       <x>352</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive15">
     <property name="geometry">
      <rect>
       <x>372</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDrive16">
     <property name="geometry">
      <rect>
       <x>392</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>11</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>12</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>13</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>14</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>15</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>16</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSelectAllDrives">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDeselectAllDrives">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Multi select</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="flyfi.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClearDebug</sender>
   <signal>clicked()</signal>
   <receiver>edtDebug</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>725</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>701</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClearError</sender>
   <signal>clicked()</signal>
   <receiver>edtError</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>725</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>712</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
